engine/apps/base/messaging.py

```diff
@@ -10,6 +10,7 @@ class BaseMessagingBackend:
 
     templater = None
     template_fields = ("title", "message", "image_url")
+    skip_default_template_fields = False
```

Comment:
I added this to avoid having these auto-added as the default template values:
<img width="1046" alt="Screenshot 2024-02-08 at 15 44 48" src="https://github.com/grafana/oncall/assets/9406895/331e77cb-81c4-41b6-af46-6d768038b69d">
(this is a screenshot of the Web templates but the same values were showing up by default for Mobile app templates (even though the templates for these fields in the `alert_receive_channel.messaging_backends_templates` object were not set))